src/model/profile.ts

```diff
@@ -0,0 +1,32 @@
+import mongoose, { Schema, Document } from 'mongoose';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * @exports
+ * @interface IDocumentProfile
+ * @extends {Document}
+ */
+
```

Comment:
Remove this empty link please, there are no empty link between document block and implementation